pkg/question_generator/questions.go

```diff
@@ -0,0 +1,28 @@
+package question_generator
+
+import (
+	"github.com/crisBrunette/questioncicas/internal/database"
+)
+
+type Question struct {
+	UUID     int
```

Comment:
You can just call this ID.
UUID is a specific type of ID, but you don't really need to use the name of the specific implementation of ID.
Generally you just want the field to be named ID since it's simpler and obvious on what it means without going too much into details of how it's implemented.